tools/syz-trace2syz/config/constants.go

```diff
@@ -0,0 +1,222 @@
+package config
+
+var (
+
+	// SpecialConsts contains values for flags found in strace. Some of these have different names in Syzkaller
+	// and others are not yet added to Syzkaller.
+	SpecialConsts = map[string]uint64{
```

Comment:
I think we need either eliminate this entirely, or at least try to minimize as much as possible and document exact reasons some of these consts are still needed.

I've looked at some consts:
1. `PROT_NONE` and `FALLOC_FL_ZERO_RANGE` are present in syzkaller consts and have the same values.
2. `USR1`/`BUS`/etc: if strace gives them different names it would be better to have a mapping from strace names to actual constant names as used in source code (and in syzkaller), e.g.
```
// strace invents own const names, map them to real const names.
constMap = map[string]string{
    "USR1": "SIGUSR1",
   ...
}
```
and then we could first apply this mapping and then lookup then in syzkaller consts. This will also help with the following point.
3. Lots of ioctl consts, e.g. `FS_IOC_GETFLAGS` have different values depending on arch. I guess your consts relate to amd64, but we don't want to limit this to amd64. Using syzkaller machinery would help with this.
4. Re missing consts, we should add them to syzkaller descriptions. E.g. you can do:
```
_ = FUTEX_WAIT_PRIVATE, FUTEX_WAKE_PRIVATE, ...
```
and them make extract should add these consts to .const files.